src/spacegame3/SpaceGame.java

```diff
@@ -80,29 +94,45 @@ public void giveSceneTo(SizableScene scene){
         stage.setScene(scene);
     }
 
+    /*
+        *main method - runs + launches game
+    */
```

Comment:
use /** */ for methods (just repeating for each one)

One interesting piece of info that can be added to the description of the main method is the list of arguments it can take (when the method takes some...)